align_system/utils/alignment_utils.py

```diff
@@ -93,10 +97,10 @@ def __call__(self, kdma_values, target_kdmas, misaligned=False):
                 distance += _euclidean_distance(target_kdma['value'], average_score)
             distances.append(distance)
 
-        selected_choice, probs = self._select_min_dist_choice(choices, distances, misaligned)
+        selected_choice, probs = self._select_min_dist_choice(choices, distances, misaligned, probabilistic=probabilistic)
         return selected_choice, probs
 
-    def get_best_sample_index(self, kdma_values, target_kdmas, selected_choice, misaligned=False):
+    def get_best_sample_index(self, kdma_values, target_kdmas, selected_choice, misaligned=False, **kwargs):
```

Comment:
Why was `**kwargs` added here?

Reply:
Whoops, removed in [7f146b3](https://github.com/ITM-Kitware/align-system/pull/106/commits/7f146b39609076b556f247f42c774ce6863d2269)